### PR TITLE
Translate mail.md

### PR DESCRIPTION
**Pull request by odie1111**
- 最後一小段的原文和 5.1 略有不同，修正譯文。
- 將 message builder 由「訊息產生器」改翻為「訊息建構器」。
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/laravel-taiwan/docs/pull/307%23issuecomment-171560021%22%2C%20%22https%3A//github.com/laravel-taiwan/docs/pull/307%23discussion_r48395469%22%2C%20%22https%3A//github.com/laravel-taiwan/docs/pull/307%23discussion_r48395472%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/laravel-taiwan/docs/pull/307%23issuecomment-171560021%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%5Cu5c07%5Cu300c%5Cu672c%5Cu5730%5Cu300d%5Cu4fee%5Cu6539%5Cu70ba%5Cu300c%5Cu672c%5Cu6a5f%5Cu300d%5Cu3002%22%2C%20%22created_at%22%3A%20%222016-01-14T07%3A17%3A21Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/4908385%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/odie1111%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%2067a6a4c5601c2821c8713dd7f63f4aa14b0a5bff%20mail.md%20258%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/laravel-taiwan/docs/pull/307%23discussion_r48395472%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%5Cu672c%5Cu5730%20%3D%3E%20%5Cu672c%5Cu6a5f%22%2C%20%22created_at%22%3A%20%222015-12-24T04%3A32%3A54Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/8567270%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/jigsawye%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20mail.md%3AL213-230%22%7D%2C%20%22Pull%2067a6a4c5601c2821c8713dd7f63f4aa14b0a5bff%20mail.md%2024%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/laravel-taiwan/docs/pull/307%23discussion_r48395469%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%5Cu672c%5Cu5730%20%3D%3E%20%5Cu672c%5Cu6a5f%22%2C%20%22created_at%22%3A%20%222015-12-24T04%3A32%3A42Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/8567270%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/jigsawye%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20mail.md%3AL1-62%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull 67a6a4c5601c2821c8713dd7f63f4aa14b0a5bff mail.md 24'></a> &nbsp;<b><a href='https://github.com/laravel-taiwan/docs/pull/307#discussion_r48395469'>File: mail.md:L1-62</a></b>
- <a href='https://github.com/jigsawye'></a> 本地 => 本機
- [ ] <a href='#crh-comment-Pull 67a6a4c5601c2821c8713dd7f63f4aa14b0a5bff mail.md 258'></a> &nbsp;<b><a href='https://github.com/laravel-taiwan/docs/pull/307#discussion_r48395472'>File: mail.md:L213-230</a></b>
- <a href='https://github.com/jigsawye'></a> 本地 => 本機
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/laravel-taiwan/docs/pull/307#issuecomment-171560021'>General Comment</a></b>
- <a href='https://github.com/odie1111'></a> 將「本地」修改為「本機」。

<a href='https://www.codereviewhub.com/laravel-taiwan/docs/pull/307?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/laravel-taiwan/docs/pull/307?approve=1'></a>&nbsp;<a href='https://github.com/laravel-taiwan/docs/pull/307'></a>
<a href='#crh-end'></a>

### CODE REVIEW

File: mail.md [hunk 1]
```diff
@@ -1,61 +1,61 @@
...
+    - [隊列郵件](#queueing-mail)
+- [郵件與本機端開發](#mail-and-local-development)
+- [事件](#events)
 
 <a name="introduction"></a>
-## Introduction
+## 簡介
 
-Laravel provides a clean, simple API over the popular [SwiftMailer](http://swiftmailer.org) library. Laravel provides drivers for SMTP, Mailgun, Mandrill, Amazon SES, PHP's `mail` function, and `sendmail`, allowing you to quickly get started sending mail through a local or cloud based service of your choice.
+Laravel 基於熱門的 [SwiftMailer](http://swiftmailer.org) 函式庫提供了一個簡潔的 API。Laravel 為 SMTP、Mailgun、Mandrill、Amazon SES、PHP 的 `mail` 函式及 `sendmail` 提供驅動，讓你可以快速地以所選擇的本地或雲端服務開始寄送郵件。
```
Comment (jigsawye) [inline mail.md]: 本地 => 本機

File: mail.md [hunk 2]
```diff
@@ -213,17 +213,17 @@ If you wish to specify a specific queue on which to push the message, you may do
...
+## 郵件與本機端開發
 
-When developing an application that sends e-mail, you probably don't want to actually send e-mails to live e-mail addresses. Laravel provides several ways to "disable" the actual sending of e-mail messages.
+當開發需要寄送電子郵件的應用程式時，你有可能不想要實際地送出電子郵件到真正的郵件地址。Laravel 提供了幾種方法以「停止」將電子郵件訊息真正寄出。
 
-#### Log Driver
+#### 日誌驅動
 
-One solution is to use the `log` mail driver during local development. This driver will write all e-mail messages to your log files for inspection. For more information on configuring your application per environment, check out the [configuration documentation](/docs/{{version}}/installation#environment-configuration).
+一個解決方案是在本地端開發時使用 `log` 郵件驅動。此驅動會將所有的電子郵件訊息寫入日誌檔案作為檢驗之用。若需要更多根據環境來設定應用程式的資訊，可參考[設定文件](/docs/{{version}}/installation#environment-configuration)。
```
Comment (jigsawye) [inline mail.md]: 本地 => 本機

**Comment by odie1111** [comment]
將「本地」修改為「本機」。

**Comment by neighborhood999** [comment]
@odie1111 can you solve conflicts？

**Pull request closed without merging**